.travis.yml

```diff
@@ -68,6 +68,24 @@ matrix:
         - WITH_MEMCHECK="ON"
         - NIFTI_SHELL_SCRIPT_TESTS="OFF"
         - SCANBUILD_EXE=""
+    - compiler: gcc
+      os: linux
+      env:
+        - BLDPREFIX=with_applications_
+        - PARALLEL_LEVEL=3
+        - AGENT_BUILDDIRECTORY=${TRAVIS_BUILD_DIR}/NIFTIworkspace
+        - BUILD_SOURCESDIRECTORY=${TRAVIS_BUILD_DIR}
+        - SYSTEM_PULLREQUEST_SOURCEBRANCH=${TRAVIS_PULL_REQUEST_BRANCH}
+        - BUILD_SOURCEBRANCHNAME=${TRAVIS_PULL_REQUEST_BRANCH}
+        - BUILD_BUILDID=${TRAVIS_BUILD_ID}
+        - SYSTEM_PULLREQUEST_PULLREQUESTNUMBER=${TRAVIS_PULL_REQUEST}
+        - CTEST_SCRIPT_DIRECTORY=${TRAVIS_BUILD_DIR}/cmake
+       - INTERPROCEDURAL_OPTIMIZATION="ON"
```

Comment:
Super minor, and does not need to be fixed, but this is not indented the same as surrounding items.